src/hooks/use-element-should-close.js

```diff
@@ -63,7 +63,9 @@ export function useElementShouldClose(closeableEl, isOpen, handleClose) {
       document.body,
       ['focus'],
       event => {
-        if (!closeableEl.current.contains(/** @type {Node} */ (event.target))) {
+        if (
+          !closeableEl.current?.contains(/** @type {Node} */ (event.target))
+        ) {
```

Comment:
To be safer and clearer about intent, what do you think about this:

```suggestion
        if (
          closeableEl.current &&
          !closeableEl.current.contains(/** @type {Node} */ (event.target))
        ) {
```

Reply:
`!closeableEl.current?.contains(/** @type {Node} */ (event.target))` is equivalent to `closeableEl.current === null || !closeableEl.current.contains(/** @type {Node} */ (event.target))`.

New behaviour: when the `closeableEl` is _not_ 'defined', invoke `handleClose` when click (or focus) events comes from _any_ element on the page.

The previous behaviour does not change: when `cloeableEl` is 'defined', invoke `handleClose` only when the click (or focus) event comes from an element _outside_ the `closeableEl`.

Reply:
> New behaviour: when the closeableEl is not 'defined', invoke handleClose when click (or focus) events comes from any element on the page.

I'm getting more and more confused. Let's use voices.